src/main/java/seedu/guest/commons/util/StringUtil.java

```diff
@@ -38,6 +38,42 @@ public static boolean containsWordIgnoreCase(String sentence, String word) {
                 .anyMatch(preppedWord::equalsIgnoreCase);
     }
 
+    /**
+     * Returns true if the {@code sentence} contains the {@code date}.
+     *   Ignores hyphen and space, but a full date match is required.
+     *   <br>examples:<pre>
+     *       containsDateIgnoreHyphenIgnoreSpace("13/09/22 - 15/09/22", "13/09/22 - 15/09/22") == true
+     *       containsDateIgnoreHyphenIgnoreSpace("13/09/22 - 15/09/22", "13/09/22 15/09/22") == true
+     *       containsDateIgnoreHyphenIgnoreSpace("13/09/22 - 15/09/22", "13/09/22") == true
+     *       containsDateIgnoreHyphenIgnoreSpace("13/09/22 - 15/09/22", "15/09/22") == true
+     *       containsDateIgnoreHyphenIgnoreSpace("13/09/22 - 15/09/22", "13/09/22 - 19/09/22")
+     *              == false //not a full date match
```

Comment:
Can we remove the testcases with more than 1 date as discussed?

Reply:
Agreed and changed.